src/js/jsx/search/SearchBar.jsx

```diff
@@ -45,7 +45,8 @@ define(function (require, exports, module) {
      * @const
      * @type {string}
      */
-    var PLACEHOLDER_ID = "NO_OPTIONS-placeholder";
+    var PLACEHOLDER_ID = "NO_OPTIONS-placeholder",
+        ADOBE_STOCK_FILTER = "ADOBE_STOCK";
```

Comment:
This is OK I guess, but it's kind of a bummer that you are directly tying one particular search provider into the search bar. It would be nicer if different search providers could optionally register "default" handlers. The default handler, in this case, would execute the stock search. It would also take the place of the "no results found" message. If I haven't explained that well, let's talk more!